### ISSUE TITLE
undefined symbol _rd_kafka_position

### ISSUE THREAD

**Issue by barock19** [edenhill/librdkafka#538]
when I trying to build my Ruby C extension for librdkafka
```sh
$ gcc -I/Users/barock/.rvm/rubies/ruby-2.1.5/include/ruby-2.1.0 \
-I/Users/barock/.rvm/rubies/ruby-2.1.5/include/ruby-2.1.0/x86_64-darwin14.0 \
-I/Users/barock/Projects/confluent-2.0.0/src/librdkafka-0.9.0_2.0.0 \
main.c -lrdkafka
```
I got this warning
```sh
main.c:233:5: warning: implicit declaration of function 'rd_kafka_position' is invalid in C99 [-Wimplicit-function-declaration]
    rd_kafka_position(rk, topics, 100);
    ^
1 warning generated.
Undefined symbols for architecture x86_64:
  "_rd_kafka_position", referenced from:
      _main in main-e0deb6.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

any suggestion ?

**Comment by edenhill** [comment]
Ruby bindings, exciting!

It looks like you are compiling against an older version of librdkafka, is it possible that rdkafka.h is installed somewhere in the system include path, e.g. /usr/include/librdkafka/rdkafka.h?

**Comment by barock19** [comment]
Oh man .. really sorry this is my bad `rdkafka` installed in my default 'include path' is version 0.9.0 using homebrew. I thought `_rd_kafka_position` is included start from 0.9.0, but when I see the `rdkafka.h` it's not there. again thanks man :smile:

**Comment by edenhill** [comment]
:+1:
